build-system/tasks/runtime-test/index.js

```diff
@@ -513,7 +513,7 @@ async function runTests() {
       if (!argv.saucelabs && !argv.saucelabs_lite) {
         log(green('Running tests locally...'));
       }
-    }).on('browsers_ready', function() {
+    }).once('browsers_ready', function() {
```

Comment:
This doesn't seem related to the PR description?

Reply:
It is mentioned in the description.

Reply:
So it is! Missed it even though I read the description twice >__>

Reply:
Removed code and edited description.

(For context, that fix was for a rare bug where this event would sometimes trigger twice, and cause the `Done. Running tests...` message to be printed more than once.)